components/base/kinematicbase/ptgKinematics.go

```diff
@@ -167,7 +190,10 @@ func (ptgk *ptgBaseKinematics) Kinematics() referenceframe.Frame {
 func (ptgk *ptgBaseKinematics) CurrentInputs(ctx context.Context) ([]referenceframe.Input, error) {
 	ptgk.inputLock.RLock()
 	defer ptgk.inputLock.RUnlock()
-	return ptgk.currentState.currentInputs, nil
+
+	planningFrameInputs := ptgk.currentState.currentInputs
+	planningFrameInputs = append(planningFrameInputs, []referenceframe.Input{}...)
```

Comment:
This seems useless?

Reply:
you're right, it does appear to be useless.